enclave_manager/avalon_enclave_manager/kme/kme_enclave_manager.py

```diff
@@ -79,25 +79,29 @@ def _manager_on_boot(self):
         self._wo_kv_delegate.cleanup_work_orders()
 
 # -------------------------------------------------------------------------
-
-    def _execute_work_order(self, input_json_str, indent=4):
+    def _execute_work_order(self, input_json_str, ext_data="", indent=4):
         """
         Submits request to KME and retrieves the response
 
         Parameters :
-            input_json_str - A JSON formatted str of the request to execute
+            @param input_json_str - A JSON formatted str of the request
+            @param ext_data - Extended data to pass to trusted code
+            @param indent - Indentation for JSON. Defaults to 4
```

Comment:
Is indent parameter a configurable in config file. If not, I'm not sure why we need it as a argument to this function as at some place it's hard coded either by caller or have to be passed to functions at all levels to maintain uniformity. I would rather hardcode this when we call json.dumps()